installer/launch-linux.sh

```diff
@@ -1,4 +1,13 @@
-#!/bin/sh
+#!/bin/bash
+
+# Query the current display DPI
+#   This is to fix the AppImage, and scale the UI correctly
+#   for high DPI displays.
+regex="^.*resolution:\s*([0-9]*)x"
+DPI_OUTPUT=$(xdpyinfo | grep -B 2 resolution)
+[[ $DPI_OUTPUT =~ $regex ]]
+DPI=$(bc <<<"scale=2;${BASH_REMATCH[1]}/92")
```

Comment:
@jonoomph 

Where did the number `92` come from, in this equation? 

Standard desktop DPI is 96, and exactly what my (and most) unscaled displays will report. So now the Linux AppImage is setting the scaling factor to `1.04` on my machine.